tests/acceptance/test_organization_developer_settings.py

```diff
@@ -34,3 +34,57 @@ def test_create_new_integration(self):
             self.browser.wait_until('.ref-success')
 
             assert self.browser.find_element_by_link_text('Tesla')
+
+
+class OrganizationDeveloperSettingsEditAcceptanceTest(AcceptanceTestCase):
+    """
+    As a developer, I can edit an existing integration
+    """
+
+    def setUp(self):
+        super(OrganizationDeveloperSettingsEditAcceptanceTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Tesla',
+            owner=self.user,
+        )
+        self.team = self.create_team(organization=self.org, name='Tesla Motors')
+        self.project = self.create_project(
+            organization=self.org,
+            teams=[self.team],
+            name='Model S',
+        )
+        self.sentry_app = self.create_sentry_app(
+            name='Tesla App',
+            organization=self.org,
+            schema={'elements': [self.create_issue_link_schema()]}
+        )
+        self.login_as(self.user)
+
+        self.org_developer_settings_path = u'/settings/{}/developer-settings/{}'.format(
+            self.org.slug, self.sentry_app.slug)
+
+    def load_page(self, url):
+        self.browser.get(url)
+        self.browser.wait_until_not('.loading-indicator')
+
+    def test_edit_integration_schema(self):
+        with self.feature('organizations:sentry-apps'):
+            self.load_page(self.org_developer_settings_path)
+
+            textarea = self.browser.element('textarea[name="schema"]')
+            textarea.clear()
+            textarea.send_keys('{}')
+
+            self.browser.click('[aria-label="Save Changes"]')
+
+            self.browser.wait_until('.ref-success')
+
+            self.browser.click('[data-test-id="tesla-app"]')
+            link = self.browser.find_element_by_link_text('Tesla App')
+            link.click()
```

Comment:
@MeredithAnya Do we need to do these two different clicks? To me, it seems we only need one of them.

Reply:
goooooood point